readthedocs/core/symlink.py

```diff
@@ -144,26 +144,26 @@ def symlink_cnames(self, domain=None):
         if domain:
             domains = [domain]
         else:
-            domains = Domain.objects.filter(project=self.project)
+            domains = [dom.domain for dom in Domain.objects.filter(project=self.project)]
```

Comment:
You can use `.values_list('domain', flat=True)` as @humitos mentioned above, no need to do comprehension here.

Reply:
Thanks for guiding me.
I have implemented it.